index.js

```diff
@@ -4,17 +4,23 @@ const fs = require('fs')
 const tablemark = require('tablemark')
 const isValidPath = require('is-valid-path')
 
+var jsonIsArray = /^\s*\[/
+var isEmpty = /^\s*$/
+
 module.exports = (path, input, options) => {
   options = Object.assign({}, options)
 
   if (path && !isValidPath(path)) {
     throw new TypeError('Invalid file path')
   }
 
-  let fileContents = path ? read(path) : input
-  let parsedInput = parse(fileContents)
+  let json = path ? read(path) : input
+  let data = parse(String(json))
+
+  if (data.length === 0)
```

Comment:
We can put this one on a single line.

```js
if (data.length === 0) return ''
```

Reply:
What's your rule on that? Single without braces or multiple with braces?

Reply:
If it's a short return like this or a simple expression I do single line. I'll pretty much always put assignments in curly braces.